nx-X11/lib/X11/Xlibint.h

```diff
@@ -28,25 +46,6 @@ from The Open Group.
 
 */
 
-/**************************************************************************/
-/*                                                                        */
-/* Copyright (c) 2001, 2011 NoMachine, http://www.nomachine.com/.         */
-/*                                                                        */
-/* NX-X11, NX protocol compression and NX extensions to this software     */
-/* are copyright of NoMachine. Redistribution and use of the present      */
-/* software is allowed according to terms specified in the file LICENSE   */
-/* which comes in the source distribution.                                */
-/*                                                                        */
-/* Check http://www.nomachine.com/licensing.html for applicability.       */
-/*                                                                        */
-/* NX and NoMachine are trademarks of Medialogic S.p.A.                   */
```

Comment:
Hmm, you removed the trademark stuff here, but the trademark seem to be still valid:  https://trademarks.justia.com/783/93/nx-78393680.html